Update Fablab_text_to_excel.py

abilvj2 opened the pull request:
Updated the steps for naming the folders with timestamp in order to create folders iteratively named and which will aid finding files with its respective histories.

abilvj2 committed:
Update Fablab_text_to_excel.py

nelson123-lab reviewed (changes requested):
I think the same will be applicable for the else condition.

abilvj2 committed:
Update Fablab_text_to_excel.py

As per the requested changes, added instructions for naming convention in the case of folder not existing.

nelson123-lab commented:
Now I believe that the above issue you mentioned can be resolved.
